### DISCUSSION TITLE
2022 Q2 블로그 결산 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#20] [Announcements]
# 2022 Q2 블로그 결산 | 정현수 기술 블로그

블로그 배포 3개월이 지난 순간 해보는 결산

https://junghyeonsu-dev.vercel.app/posts/2022-Q2#comments-container

**Comment by daheejo** [comment]
벨로그 -> 티스토리 타고 블로그 만드신 것에 대한 포스팅을 쭉 봤는데 정말 대단하십니다!! 트래픽 천천히 곧 많아질 것이라 예상합니다! :)

**Reply by junghyeonsu** [reply]
글 읽어주셔서 감사합니다!.. 좋게 봐주셔서 감사해요 ㅎㅎㅎㅎ

**Comment by ch0213** [comment]
안녕하세요~

**Reply by junghyeonsu** [reply]
안녕하세요^_^
